core/src/CMakeLists.txt

```
@@ -164,7 +169,13 @@ if(Kokkos_ENABLE_IMPL_MDSPAN)
       PATTERN "mdspan"
       PATTERN "*.hpp"
     )
-    message(STATUS "Using internal mdspan directory ${KOKKOS_SOURCE_DIR}/tpls/mdspan/include")
+
+    set(KOKKOS_INTERNAL_MDSPAN_VERSION "kokkos/mdspan@0e4eba59ad53451784b3839bdf98b25fa2d64931")
+    global_set(KOKKOS_MDSPAN_VERSION "internal (${KOKKOS_INTERNAL_MDSPAN_VERSION})")
+    message(
+      STATUS
+        "Using internal mdspan copy (${KOKKOS_INTERNAL_MDSPAN_VERSION})"
+    )
```

Sorry the format was wrong
```suggestion
    message(STATUS "Using internal mdspan copy (${KOKKOS_INTERNAL_MDSPAN_VERSION})")
```